### ISSUE TITLE
Bug: react-devtools doesn't install on macs with the M1 chip because it has an incompatible version of electron

### ISSUE THREAD

**Issue by coreyar** [facebook/react#21210]
React version: 16.13.1
react-devtools 4.10.0

## Steps To Reproduce

1. Install react-devtools 4.10.0 on a mac with an M1 chip
2. The installation will fail becauase electron 9.4.4 is incompatible 

Link to code example:
This commit has a version bump for electron but it looks like the package was never republished because the react-devtools package version wasn't updated
https://github.com/facebook/react/commit/beb38aba3e5617cb915a8e979245b2996e1764e8#diff-643298d1c50c9333dd9f9b34d4554595c61a68758147effab85d58330772e8a1

## The current behavior
```
error /~/node_modules/react-devtools/node_modules/electron: Command failed.
Exit code: 1
Command: node install.js
Arguments:
Directory: /~/node_modules/react-devtools/node_modules/electron
Output:
HTTPError: Response code 404 (Not Found) for https://github.com/electron/electron/releases/download/v9.4.4/electron-v9.4.4-darwin-arm64.zip
    at EventEmitter.<anonymous> (/~/node_modules/got/source/as-stream.js:35:24)
    at EventEmitter.emit (node:events:369:20)
    at module.exports (/~e/node_modules/got/source/get-response.js:22:10)
    at ClientRequest.handleResponse (/~/node_modules/got/source/request-as-event-emitter.js:155:5)
    at Object.onceWrapper (node:events:476:26)
    at ClientRequest.emit (node:events:381:22)
    at ClientRequest.origin.emit (/~node_modules/@szmarczak/http-timer/source/index.js:37:11)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (node:_http_client:646:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:129:17)
    at TLSSocket.socketOnData (node:_http_client:512:22)
```
## The expected behavior
Installs without errors

**Comment by bvaughn** [comment]
This was already reported in #20454 and fixed via #20496. It just hasn't been published to NPM yet.

**Comment by coreyar** [comment]
ok thanks any timeline on publishing it?

**Comment by bvaughn** [comment]
I'm going to try to do it this morning. :)

**Comment by bvaughn** [comment]
published as 4.11.0
